espnet2/bin/slu_inference.py

```diff
@@ -272,6 +273,58 @@ def __call__(
         speech = speech.unsqueeze(0).to(getattr(torch, self.dtype))
         # lengths: (1,)
         lengths = speech.new_full([1], dtype=torch.long, fill_value=speech.size(1))
+
+        if self.run_chunk:
+            speech = to_device(speech, device=self.device)
+            sim_chunk_length = 640
+            start_chunk = 3200
```

Comment:
It's better to make them controlled by arguments.

Reply:
Done!